man/qmethod.Rd

```diff
@@ -20,11 +42,19 @@ qmethod(dataset, nfactors, rotation = "varimax", forced = TRUE,
+If the input data contains row names and variable names, these will be kept throughout the analysis. Input data is validated, and it will give an error if there are non-numerical values or either if the number of statements or Q-sorts introduced does not match the input data. It also returns an error if the argument \code{forced} is set to \code{TRUE} but Q-sorts contain differing distributions.
+
+  Unrotated \emph{principal} components are always in decreasing order of the sum of squared loadings, also known as Eigenvalues.
+  Rotating the loadings matrix, for example with \code{rotation = "varimax"}, redistributes the loadings among the components and the resultant sum of squared loadings (no longer Eigenvalues) may no longer be in a decreasing order.
+  By convention and for convenience, some factor extraction and rotation routines such as \code{\link[psych]{principal}} \emph{re-order} such rotated components according to their sum of squared \emph{rotated} loadings.
+  Such re-ordering does \emph{not} change the results, but merely the presentation and is generally discouraged for Q methodology, where the relative sum of squared loadings are generally considered less informative (compare Brown 1980: 223).
+  Re-ordering is also incompatible with \code{\link{q.mrot.do}} and \code{\link{q.mrot.choose}} for by-hand rotation.
+  \code{reorder = TRUE} is available as an option for backward compatibility: prior to version \code{"1.4.0"} \code{\link{qmethod}} \emph{did} reorder components, and the presentation of results may differ as a result in rare cases.
+  Notice that \code{\link{qmethod}} still inverts components that load negative on average by multiplying them with \code{-1}, as is also default behavior for \code{\link[psych]{principal}}.
 }
 \value{
 Returns a list of class \code{QmethodRes}, with eight objects:
-\item{brief}{a list with the basic values of the analysis: date (\code{"date"}), number of statements (\code{"nstat"}), number of Q-sorts (\code{"nqsort"}), whether the distribution was 'forced' (\code{"distro"}),  number of factors extracted (\code{"nfactors"}), type or rotation (\code{"rotation"}), method for correlation in the PCA (\code{"cor.method"}), and a summary of this information for display purposes (\code{"info"}).}
+\item{brief}{a list with the basic values of the analysis: date (\code{"date"}), number of statements (\code{"nstat"}), number of Q-sorts (\code{"nqsort"}), whether the distribution was 'forced' (\code{"distro"}), number of factors extracted (\code{"nfactors"}), type or rotation (\code{"rotation"}), whether components were re-ordered on rotation (\code{"reorder"}), method for correlation in the PCA (\code{"cor.method"}), a summary of this information for display purposes (\code{"info"}), the rotation matrix (\code{"rotmat"}) and how Q-sorts were flagged (\code{"flagging"}).}
```

Comment:
change to "the #manual# rotation matrix"

Is this in the same order as the elements in 'brief'?